core/http-auth-aws/src/main/java/software/amazon/awssdk/http/auth/aws/util/CredentialUtils.java

```diff
@@ -13,14 +13,14 @@
  * permissions and limitations under the License.
  */
 
-package software.amazon.awssdk.http.auth.internal.util;
+package software.amazon.awssdk.http.auth.aws.util;
 
-import software.amazon.awssdk.annotations.SdkInternalApi;
+import software.amazon.awssdk.annotations.SdkProtectedApi;
 import software.amazon.awssdk.identity.spi.AwsCredentialsIdentity;
 import software.amazon.awssdk.identity.spi.AwsSessionCredentialsIdentity;
 import software.amazon.awssdk.utils.StringUtils;
 
-@SdkInternalApi
+@SdkProtectedApi
```

Comment:
Any way to avoid this needing to be protected? Or can we reduce the number of things it contains to strictly what is needed to be protected?

Reply:
I guess we can make things that aren't used cross-module package-private?

Reply:
We can also get rid of this entirely if it's a problem - as long as duplication is fine where this needs to be used (once in this module, and likely in CRT).